src/constructs/ecs/ecs-task.ts

```diff
@@ -144,7 +145,7 @@ export class GuEcsTask {
       // see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecs-taskdefinition.html#cfn-ecs-taskdefinition-cpu for details
       cpu = 2048, // 2 cores and from 4-16GB memory
       memory = 4096, // 4GB
-
+      storage = 20, //20 GB
```

Comment:
This is an invalid input. Valid value for `ephemeralStorageGiB` are:

* Undefined (which will default to 20 GiB).
* Between 21 and 200 (inclusive).